TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Security/Authorization/Privilege/Node/AbstractNodePropertyPrivilege.php

```diff
@@ -47,7 +47,7 @@
     public function matchesSubject(PrivilegeSubjectInterface $subject)
     {
         if ($subject instanceof PropertyAwareNodePrivilegeSubject === false && $subject instanceof MethodPrivilegeSubject === false) {
-            throw new InvalidPrivilegeTypeException(sprintf('Privileges of type "TYPO3\TYPO3CR\Security\Authorization\Privilege\Node\ReadNodePropertyPrivilege" only support subjects of type "TYPO3\TYPO3CR\Security\Authorization\Privilege\Node\PropertyAwareNodePrivilegeSubject" or "TYPO3\Flow\Security\Method\MethodPrivilegeSubject", but we got a subject of type: "%s".', get_class($subject)), 1417018448);
+            throw new InvalidPrivilegeTypeException(sprintf('Privileges of type "'.ReadNodePropertyPrivilege::class.'" only support subjects of type "'.PropertyAwareNodePrivilegeSubject::class.'" or "'.MethodPrivilegeSubject::class.'", but we got a subject of type: "%s".', get_class($subject)), 1417018448);
```

Comment:
As this is already using sprintf we should make the classes arguments now or at least correctly add spaces between the concatenation.